src/setup.py

```diff
@@ -0,0 +1,17 @@
+from setuptools import setup, find_packages
+
+setup(
+    name="src",
+    version="0.1.0",
+    install_requires=[
```

Comment:
We should probably set the `packages` arg explicitly, eg. `packages=["src"]`, as there are py files elsewhere in the repo.

Reply:
change made